server/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import { createExpressApp } from './server';
+import { getPort, getTwilioCredentials } from './utils/env';
+import { log, warn } from './utils/log';
+
+async function main() {
+  log('Ahoy! Starting Twilio Voice React Native Reference server...');
+
+  const twilioCredentials = getTwilioCredentials();
+  if (typeof twilioCredentials === 'undefined') {
+    warn(
+      'Incomplete Twilio Credentials, please validate your environment ' +
+      'variables.'
+    );
+    return;
+  }
+
+  const port = getPort() ?? 3003;
```

Comment:
nit: Can we use 3000 or 3030? Just trying to be consistent with other node apps.